docs/content/connectors/oceanbase-cdc(ZH).md

```diff
@@ -0,0 +1,285 @@
+# OceanBase CDC 连接器
+
+OceanBase CDC 连接器允许从 OceanBase 读取快照数据和增量数据。本文介绍了如何设置 OceanBase CDC 连接器以对 OceanBase 进行 SQL 查询。
+
+## 依赖
+
+为了使用 OceanBase CDC 连接器，您必须提供相关的依赖信息。以下依赖信息适用于使用自动构建工具（如 Maven 或 SBT）构建的项目和带有 SQL JAR 包的 SQL 客户端。
+
+```xml
+<dependency>
+  <groupId>com.ververica</groupId>
+  <artifactId>flink-connector-oceanbase-cdc</artifactId>
+  <!--  请使用已发布的版本依赖，snapshot 版本的依赖需要本地自行编译。 -->
+  <version>2.3.0</version>
+</dependency>
+```
+
+## 下载 SQL 客户端 JAR 包
+
+点击 [flink-sql-connector-oceanbase-cdc-2.3.0.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-oceanbase-cdc/2.3.0/flink-sql-connector-oceanbase-cdc-2.3.0.jar) 下载 JAR 包至 `<FLINK_HOME>/lib/`.
+
+> **说明：**
+>
+> 下载链接仅适用于稳定发行版本。
+
+`flink-sql-connector-oceanbase-cdc-XXX-SNAPSHOT` 快照版本与开发分支的版本对应。要使用快照版本，您必须自行下载并编译源代码。推荐使用稳定发行版本，例如 `flink-sql-connector-oceanbase-cdc-2.3.0.jar`。您可以在 Maven 中央仓库中找到使用稳定发行版本。
```

Comment:
Users need to download the source code and compile the corresponding jar. Users should use the released version, such as [flink-sql-connector-oceanbase-cdc-2.3.0.jar](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-oceanbase-cdc), the released version will be available in the Maven central warehouse.